Lombiq.HelpfulLibraries.Common/Validation/EmailValidationHelpers.cs

```diff
@@ -5,10 +5,16 @@
 namespace Lombiq.HelpfulLibraries.Common.Validation;
 
 /// <summary>
-/// Helpers to validate e-mail addresses.
+/// Helpers to validate email addresses.
```

Comment:
Why change this everywhere?

Reply:
To standardize it in both code and documentation -- in code it's treated as one word ("`IsValidEmailAddress()`" instead of "`IsValidEMailAddress()`"), which is better represented by the non-dash version I think. Trivial, to be sure, but I found it preferable this way.

Reply:
I see, OK then.